snap/snapcraft.yaml

```diff
@@ -223,48 +223,51 @@ parts:
     plugin: dump
     source: https://zoom.us/client/latest/zoom_amd64.deb
     override-build: |
-      # remove danging symlink
-      rm -f $SNAPCRAFT_PART_BUILD/usr/bin/zoom
-      echo "QT_QPA_PLATFORM=xcb \$SNAP/opt/zoom/ZoomLauncher" > $SNAPCRAFT_PART_BUILD/usr/bin/zoom
-      chmod +x $SNAPCRAFT_PART_BUILD/usr/bin/zoom
-      # Unfrig LD_LIBRARY_PATH
-      sed -i s/D_LIBRAR/XXXXXXXX/g opt/zoom/ZoomLauncher
-      snapcraftctl build
+      if [[ `arch` = "x86_64" ]]; then
```

Comment:
Better to use `${SNAPCRAFT_ARCH_TRIPLET}` here?

Reply:
:shrug: I copied one of popey's examples. Does it matter?